oauth2client/contrib/django_util/__init__.py

```diff
@@ -276,7 +382,7 @@ def __init__(self, request, scopes=None, return_url=None):
 
         # make sure previously requested custom scopes are maintained
         # in future authorizations
-        credentials = storage.get_storage(self.request).get()
+        credentials = self.credentials
```

Comment:
done, though both are slightly painful and the purpose of 1) isnt totally clear to me.